sql/schemas/bookbrainz.sql

```diff
@@ -1055,4 +1055,121 @@ CREATE VIEW bookbrainz.work_import AS
 	LEFT JOIN bookbrainz.alias_set alias_set ON work_data.alias_set_id = alias_set.id
 	WHERE import.type = 'Work';
 
+
+INSERT INTO bookbrainz.relationship_attribute_type (id, parent, root, child_order, name, description)
```

Comment:
This file should not be changed in this case.
We modify it when we add or modify a table or something like that, but not when we add rows of data to the database, that is why we have separate migration scripts.
So you can remove all these additions in bookbrainz.sql